.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Nuvem Development Container",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:22-bookworm",
+  "postCreateCommand": "npm install"
```

Comment:
Shouldn't we run this in such a way that the lock file won't change? Like `npm ci`